auth-external-api/src/main/java/uk/gov/di/authentication/external/services/UserInfoService.java

```
@@ -82,17 +91,62 @@ private void addClaimsFromToken(
         if (accessTokenInfo.getClaims().contains(AuthUserInfoClaims.EMAIL_VERIFIED.getValue())) {
             userInfo.setEmailVerified(userProfile.isEmailVerified());
         }
-        if (accessTokenInfo.getClaims().contains(AuthUserInfoClaims.PHONE_NUMBER.getValue())) {
-            userInfo.setPhoneNumber(userProfile.getPhoneNumber());
-        }
-        if (accessTokenInfo.getClaims().contains(AuthUserInfoClaims.PHONE_VERIFIED.getValue())) {
-            userInfo.setPhoneNumberVerified(userProfile.isPhoneNumberVerified());
-        }
         if (accessTokenInfo.getClaims().contains(AuthUserInfoClaims.SALT.getValue())) {
             String base64StringFromSalt = bytesToBase64(userProfile.getSalt());
             LOG.info("is salt from UserProfile defined: {}", base64StringFromSalt != null);
             userInfo.setClaim("salt", base64StringFromSalt);
         }
+
+        var phoneDataResult = getPhoneDataIfSMSIsDefault(userProfile, userCredentials);
+
+        PhoneData phoneData;
+        if (phoneDataResult.isFailure()) {
+            LOG.warn(
+                    "Default MFA retrieval failed method found, error: {}",
+                    phoneDataResult.getFailure());
+            phoneData = new PhoneData(null, false);
```

I've decided not to error on phone being not found. Is this right, or should we return an error?